lib/cdo/hamburger.rb

```diff
@@ -10,18 +10,22 @@ class Hamburger
   HIDE_ALWAYS = "hide-always".freeze
   SHOW_MOBILE = "show-mobile".freeze
 
+  def self.get_divider_visibility(above_section_visibility, below_section_visibility)
+    return HIDE_ALWAYS if above_section_visibility == HIDE_ALWAYS || below_section_visiblity == HIDE_ALWAYS
```

Comment:
There is a typo here.

Reply:
This PR has taught me that I do not know how to spell visibility haha. Thanks I'll fix it :)